performance-storage-service/pss_project/api/models/rest/OLTPBenchRest.py

```diff
@@ -46,7 +63,8 @@ def convert_weights_to_dict(weights_list):
 def convert_metrics_to_dict(metrics):
     db_formatted_metrics = {
         'throughput': metrics.throughput,
-        'latency': metrics.latency.__dict__
+        'latency': metrics.latency.__dict__,
+        'memory_info': to_dict(metrics.memory_info),
```

Comment:
Why `to_dict()` vs `__dict__`

Reply:
Because `to_dict()` can translate an object to a Python dict recursively. If using `__dict__` here, the result would be something like
```
{'rss': <__main__.MemoryItemSummary object at 0x1040de5d0>, 'vms': <__main__.MemoryItemSummary object at 0x1040de690>}
```